generated/proto/policy.proto

```diff
@@ -34,3 +34,12 @@ message Policy {
 	Resolution resolution = 1;
 	Retention retention = 2;
 }
+
+message CompressedPolicy {
+	Policy policy = 1;
```

Comment:
My initial thinking was that it might be better to keep them separate in case we want reuse the Policy definition in a situation where we don't need id, but I don't feel too strongly either way.